core/domain/draft_upgrade_services_test.py

```diff
@@ -93,22 +93,65 @@ def test_try_upgrade_failure_due_to_unimplemented_upgrade_methods(self):
             draft_upgrade_services.try_upgrading_draft_to_exp_version(
                 self.DRAFT_CHANGELIST, 1, exploration.version, self.EXP_ID))
 
-    def test_try_upgrade_success(self):
+
+class DraftUpgradeUtilUnitTests(test_utils.GenericTestBase):
+    """Test the DraftUpgradeUtil module."""
+
+    # EXP_ID and USER_ID used to create default explorations.
+    EXP_ID = 'exp_id'
+    USER_ID = 'user_id'
+
+    # Creates and saves a default exploration.
+    def setUp(self):
+        super(DraftUpgradeUtilUnitTests, self).setUp()
+        self.save_new_valid_exploration(self.EXP_ID, self.USER_ID)
+
+    def helper_to_test_schema_verison_updates(
```

Comment:
Function names should start with verbs, and be specific. Try to avoid names like "helper" -- such names are vague and make it difficult to understand what the function actually does.

Instead, have the function name say exactly what it is doing.

Reply:
Fixed it based off of @DubeySandeep's suggestion :)